core/eval.go

```diff
@@ -87,6 +89,29 @@ func evalSET(args []string) []byte {
 				return Encode(errors.New("ERR value is not an integer or out of range"), false)
 			}
 			exDurationMs = exDurationSec * 1000
+
+		case "PXAT", "pxat":
```

Comment:
Do we need to have a check for max expiry time? What is the redis behaviour in this case? Does it have a check for max expiry time?